recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -17,6 +16,11 @@
 import conda_build.api
 import conda_build.config
 
+from conda_forge_ci_setup.feedstock_outputs import (
```

Comment:
Instead of importing like this, this should be done as in https://github.com/conda-forge/conda-forge-ci-setup-feedstock/blob/master/recipe/conda_forge_ci_setup/build_utils.py#L138-L148

Reply:
Right. It was one way or the other. Happy to go the other way.

Reply:
On second thought. Wouldn't `from .feedstock_output` work?

Reply:
It doesn't. I had that originally. I did not fully grep why it didn't work, but the error was something about the file not being in a module.

Reply:
To be clear, it complained about a relative import outside of a module.